draft/spec/index.html

```diff
@@ -180,24 +180,30 @@ <h2>Introduction</h2>
     <section id="need">
         <h2>Need</h2>
         <p>
-           The OCFL initiative arose from a need to have well-defined application-independent file management within
-           digital repositories.
+          The OCFL initiative arose from a need to have well-defined application-independent file management within
+          digital repositories.
         </p>
         <p>
-           A general observation is that the contents of a digital repository &#8212; that is, the digital files and
-           metadata that an institution might wish to manage &#8212; are largely stable. Once content has been
-           accessioned, it is unlikely to change significantly over its lifetime. This is in contrast to the software
-           applications that manage these contents, which are ephemeral, requiring constant updating and replacement.
-           Thus, transitions between application-specific methods of file management to support software upgrades and
-           replacement cycles can be seen as unnecessary and risky change which affects the long-term stable objects
-           in support of the short-term, ephemeral software.
+          A general observation is that the contents of a digital repository &#8212; that is, the digital files and
+          metadata that an institution might wish to manage &#8212; are largely stable. Once content has been
+          accessioned, it is unlikely to change significantly over its lifetime. This is in contrast to the software
+          applications that manage these contents, which are ephemeral, requiring constant updating and replacement.
+          Thus, transitions between application-specific methods of file management to support software upgrades and
+          replacement cycles can be seen as unnecessary and risky change which affects the long-term stable objects
+          in support of the short-term, ephemeral software.
         </p>
         <p>
-           By providing a specification for the file and directory layout on disk or in an object store, OCFL is an
-           attempt at reducing, or even eliminating, the need for these transitions. As an application-independent
-           specification, conforming applications will natively "understand" the underlying file structure without
-           needing to first transition these contents to their own format.
+          By providing a specification for the file and directory layout on disk or in an object store, the OCFL is an
```

Comment:
I'm not sure that we want to say "the OCFL" everywhere. I'm fine with "the OCFL initiative" but would otherwise just say "OCFL" -- we should decide about this as a style issue as a change would also have impact on the implementation notes --> #270